sys/linux/dev_vfio.txt

```diff
@@ -0,0 +1,51 @@
+# Copyright 2017 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
```

Comment:
Add a NOTE: comment here saying that we were not able to figure out how to create any groups (or what was that) and we did not get any coverage and the descriptions are incomplete.
Somebody may look at this file and think that we have good descriptions already. Checkbox. Done. syzkaller tests vfio. Nothing to do here anymore.